### PR TITLE
Map markers

### LINKED ISSUE #51: Add map markers

### PR DESCRIPTION
**Pull request by Count-MonteCristo**
Closes #51

### CODE REVIEW

File: src/components/pages/map.js
Comment (dkprioleau) [inline src/components/pages/map.js:1]: You need to make this file uppercase so the path can match what's in App.js

File: src/components/pages/map.js [hunk 1]
```diff
@@ -2,17 +2,80 @@ import React, { useEffect } from "react";
...
         'Map data © <a href="https://openstreetmap.org">OpenStreetMap</a> contributors',
     }).addTo(map);
+
+    // Add markers to the map
+    const addMarkerToMap = (lat, lng, petInfo, isFound) => {
+      const randomOffset = Math.random() * 0.018; // Adjust the offset value as needed. Helps prevent markers overlapping with one another.
+
+      const markerIcon = L.divIcon({
+        html: `<i class="bi bi-geo-alt-fill" style="color: ${
+          isFound ? "#2793E2" : "#FD678D"
```
Comment (dkprioleau) [inline src/components/pages/map.js]: nice idea to color code them

**Review by dkprioleau — APPROVED** [review]
Good work, just make sure to fix the filename before merging

File: src/components/pages/map.js
Comment (Count-MonteCristo) [reply src/components/pages/map.js:1]: Done!

File: src/components/pages/map.js (hunk 1 above)
Comment (Count-MonteCristo) [reply src/components/pages/map.js]: Thanks!

**Pull request merged**